pycbc/types/timeseries.py

```diff
@@ -844,16 +855,19 @@ def add_into(self, other):
 
         other = other.copy()
         dt = float((other.start_time - self.start_time) * self.sample_rate)
+
         if not dt.is_integer():
-            diff = (dt - _numpy.floor(dt))
+            diff = (dt - _numpy.floor(dt)) * self.delta_t
```

Comment:
This seems to change the meaning of `diff` from samples to seconds, is it intended? Was the previous code incorrect?

Reply:
@titodalcanton Yes, it does. Cyclic_time_shift takes sense, I think this was a bug. I'm not sure how I missed this before.